provider/aws.go

```diff
@@ -271,12 +269,39 @@ func changesByZone(zones map[string]*route53.HostedZone, changeSet []*route53.Ch
 	return changes
 }
 
+func expandAliasTargets(eps []*endpoint.Endpoint) (endpoints, aliasTargets []*endpoint.Endpoint) {
```

Comment:
does it expand or it splits alias from rest ?

Reply:
yeah, i guess split makes more sense